plugins/registry.ts

```diff
@@ -218,12 +217,6 @@ export default class PluginRegistry {
             data,
         });
 
-        store.dispatch({
-            type: ActionTypes.RECEIVED_PLUGIN_COMPONENT,
-            name: 'MobileChannelIntroButton',
```

Comment:
There were no other references to this name anywhere, so I think it was just copy and pasted from the `ChannelHeader` button